staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -654,7 +624,7 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 				string(types.MergePatchType),
 				string(types.StrategicMergePatchType),
 			}
-			handler := metrics.InstrumentRouteFunc(action.Verb, resource, subresource, requestScope, restfulPatchResource(patcher, reqScope, admit, mapping.ObjectConvertor, supportedTypes))
+			handler := metrics.InstrumentRouteFunc(action.Verb, resource, subresource, requestScope, restfulPatchResource(patcher, reqScope, admit, a.group.Convertor, supportedTypes))
```

Comment:
Do you know if this Convertor is the same as the convertor in the reqScope?

Reply:
If I traced it correctly, it is

Reply:
Actually same question about cross-group subresources applies